### DISCUSSION TITLE
Google play target api restrictions

### DISCUSSION THREAD

**Discussion by anantshri** [OWASP/mastg#2568] [Q&A] [answered]
Google play which is arguably the most popular app store for android applications have a strict target sdk requirements for app listing now. 

https://developer.android.com/google/play/requirements/target-sdk

This helps in curbing a large number of issues with app compatibility and security issues in older sdk's also.

I was wondering do we want to make a mention of this anywhere in the guide or we consider that this is a one store requirements and doesnt do justice to note this in the guide.

**Comment by githubrlloyd** [comment, marked as answer]
I think the intent behind the target api restrictions is to provide incentive for app developers/publishers to stay current and try and increase the adoption of newer Android features. There can indeed be security features that benefit from this requirement, but as long as the minSDK is set lower for app compatibility the security risk of older android versions remains. I would think getting the correct parameters of targetSDK, and minSDK for your app is a function of your security threat model and the compatibility requirements of your user base.

**Comment by cpholguera** [comment]
I completely agree with Ryan. We're considering minSDKversion when creating new Android V2 tests.
